README.md

```diff
@@ -467,6 +467,19 @@ describe("DockerComposeEnvironment", () => {
 });
 ```
 
+You can supply a list of service names to only start those services:
+
+```javascript
+const { DockerComposeEnvironment } = require("testcontainers");
+
+const environment = await new DockerComposeEnvironment(composeFilePath, composeFile).up(["database_service", "queue_service"]);
+
+// only database_service and queue_service will be started and available
+
+await environment.stop();
```

Comment:
nitpick: no need to re-document how to stop the environment, I think this line and the comment above can be removed